Source/Pawnmorphs/Esoteria/HPatches/PawnCompPatches.cs

```diff
@@ -103,6 +104,12 @@ private static void GiveSapientAnimalsNeeds(Pawn_NeedsTracker __instance, Pawn _
                     __result = Need_Control.IsEnabledFor(___pawn);
                     return;
                 }
+                //Need_Control.EnabledRaces.Contains(___pawn.def); Can't use it because it's private
+                var raceMutationSettingsExtension = ___pawn.kindDef.race.GetModExtension<RaceMutationSettingsExtension>();
```

Comment:
I'm wondering about the performance here, any "Get" methods tend to be relatively slow since they are seek functions.

If no other options, then I know harmony has a way to access private members with near native performance using IL injection.
(.Net Reflection is very slow)